clients/js/src/hash.ts

```
@@ -39,32 +48,106 @@ export function hashLeaf(
   return hash(
     array(getCreatorSerializer(), { size: 'remainder' }).serialize(creators)
   );
 }
+
+export function hashCollection(collection: PublicKey): Uint8Array {
+  return hash(publicKeySerializer().serialize(collection));
+}
+
+export function hashAssetData(assetData?: string | Uint8Array): Uint8Array {
+  if (assetData === undefined || assetData === null) {
+    assetData = new Uint8Array(0);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)